IO/matRad_readHLUT.m

```diff
@@ -0,0 +1,52 @@
+function hlut = matRad_readHLUT(filename)
```

Comment:
cant you reuse the equivalent function from the dicom import? I have seen that there also the particle type is required from the plan but maybe there is also a way around?

Reply:
I don't like the matRad_loadHLUT function in this context. Actually I see it more as a function to automatically determine which file to load, and thus I wanted to write a function that does nothing more than actually read the file (which is by the way now also used in matRad_loadHLUT).